### DISCUSSION TITLE
对于检测指定物体内的文字应该参考哪个模型呢？

### DISCUSSION THREAD

**Discussion by myalos** [PaddlePaddle/PaddleOCR#12409] [Q&A]
如题，比如我只检测车牌里面的字符，对于车牌外的字符就不检测
我想一种方法是先把车牌检测出来，然后再检测字符，能否一阶段直接检测车牌内的字符呢？

**Comment by SWHL** [comment]
这个一阶段做车牌识别难度较大。建议先检测出车牌，后识别
